### ISSUE TITLE
Dark themes use black text in editor window on MacOS 11

### ISSUE THREAD

**Issue by patrickserrano** [microsoft/azuredatastudio#13549]
- Azure Data Studio Version: 1.24.0
- Azure Data Studio Insiders Version: Version: 1.25.0-insider

Steps to Reproduce:

1. Update to MacOS 11 Big Sur
2. Activate a dark theme in Azure Data Studio

I've tried a number of dark themes and they all produce the same results. Switching to a light theme is a workaround, but I didn't see this reported anywhere so hopefully it's helpful.

`Dark Azure Data Studio (Default Dark)`
<img width="1024" alt="Screen Shot 2020-11-25 at 1 05 45 PM" src="https://user-images.githubusercontent.com/1302453/100265960-05a31e80-2f1f-11eb-885f-47291ecedf71.png">

`Light Azure Data Studio (Default Light)`
<img width="1024" alt="Screen Shot 2020-11-25 at 1 05 58 PM" src="https://user-images.githubusercontent.com/1302453/100265959-050a8800-2f1f-11eb-991c-1cb8a0dcf65b.png">

**Comment by aasimkhan30** [comment]
@patrickserrano is this a remote machine or a local one?

**Comment by patrickserrano** [comment]
@aasimkhan30 this is local on a 16" MacBook Pro. It looks like the issue is actually with my font Source Code Pro. I'm basing that on this issue I found today https://github.com/adobe-fonts/source-code-pro/issues/250 which I found from this VS Code issue https://github.com/microsoft/vscode/issues/100840

Installing an updated version of Source Code Pro with the SVG table removed looks to have resolved the issue so I'm going to close this. Thanks for taking a look though :) 
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/1302453/100943376-204f3780-34cb-11eb-94d8-a415c253f55b.png">
